puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/smart-proxy.groovy

```diff
@@ -0,0 +1,106 @@
+pipeline {
+    agent none
+	parameters {
+		string(name: 'ruby_versions', defaultValue: '2.5,2.6,2.7', description: 'Ruby versions to test with. Last value is used for Rubocop')
```

Comment:
I guess this isn't used right now?

Reply:
Oops, no. That was an attempt that didn't work.